Docs (GraphQL Content for Tour): Navigation and edit pass, patch-in to PR #351

aaroncarey opened the pull request:
Navigation changes, also edits to /graphqlintro/ content (more in-progress).
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/dgraph-io/tutorial/352)
<!-- Reviewable:end -->
 
 
 
<!-- Dgraph:start -->
Tour Preview: [<img src="https://bl.ocks.org/prashant-shahi/raw/3a9f99bec84231cfe3c0e82cf883f159/0e588d908ad8c8b10958b87ebdd2ba68779ccf4f/dgraph.svg" height="34" align="absmiddle" alt="Dgraph Preview"/>](https://dgraph-tour-35dfa32867-128040.surge.sh)
<!-- Dgraph:end -->

aaroncarey committed:
Update landing page, edit "Introduction" section

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/dgraph-io/tutorial?pullRequest=352) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/dgraph-io/tutorial?pullRequest=352) before we can accept your contribution.<br/>**0** out of **2** committers have signed the CLA.<br/><br/>:x: verneleem<br/>:x: aaroncarey<br/><hr/>**verneleem** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/dgraph-io/tutorial?pullRequest=352) it.</sub>

aaroncarey committed:
Fix navigation page

aaroncarey committed:
More edits, main page changes, and spellcheck

Completed edit of /graphqlintro/ and /graphqlbasic/ (1-5), also spellchecked all new files.
